docs/admin/teams/index.mdx

```diff
@@ -34,7 +34,7 @@ After hitting create, you will be redirected to the team page where you can add
 
 ### From the CLI
 
-If you prefer a command line based approach, or would like to integrate an external system of record for teams into Sourcegraph, [src-cli](https://github.com/sourcegraph/src-cli) (v5.0+) provides commands to manage teams:
+If you prefer a command line based approach, or would like to integrate an external system of record for teams into Sourcegraph, [src-cli](https://github.com/sourcegraph/src-cli) provides commands to manage teams:
```

Comment:
the docs are versioned these days, no need to call out version numbers esp if so ancient.